### PR TITLE
Use atom-package-deps to install atom-linter automatically

### PR DESCRIPTION
**Pull request by 1000ch**
ref #18

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,12 @@
     "atom": ">=1.0.0"
   },
   "dependencies": {
+    "atom-package-deps": "^2.0.1",
```
Comment (steelbrain) [inline package.json]: Please bump this to `^2.0.2`. it has a fix for windows

**Comment by 1000ch** [comment]
@steelbrain Also fixed.

**Comment by steelbrain** [comment]
LGTM Thanks!

**Comment by 1000ch** [comment]
Thank you too for nice feature `atom-package-deps`!

**Comment by steelbrain** [comment]
@1000ch The pleasure is mine :bow:

**Pull request merged**